op.h

```diff
@@ -64,6 +64,12 @@ typedef PERL_BITFIELD16 Optype;
     U8		op_private;
 #endif
 
+#define OpTYPE_set(o,type)                      \
```

Comment:
The new guidance about STMT_START { ... } STMT_END talk about how to bomb-proof such macros.  I suspect it isn't an issue here as long as long as both  o and type are not going to be expressions or have side effects.  I'm pointing this out to be sure you consider it.

Reply:
I did not verify that there were no substantial changes introduced by splitting the files.  I view it likely there weren't; so given that proviso it LGTM